### ISSUE TITLE
QSsolver find energy min always return success is False

### ISSUE THREAD

**Issue by sophietheis** [DamCB/tyssue#272]
Hi, 

I worked with a 2D circular epithelium with planar geometry. My model is composed of area elasticity, perimeter elasticity and a circular constraint. 
When I tried to get my epithlium at the equilibrium before adding any events, I always get this error : 
```python
Stop because solver didnt succeed   message: ABNORMAL_TERMINATION_IN_LNSRCH
  success: False
   status: 2
      fun: 111.35619255538515
        x: [-1.259e+02  2.038e+00 ...  7.371e+01 -4.393e+01]
      nit: 0
      jac: [ 5.421e-03  1.072e-02 ... -6.272e-02 -1.670e-03]
     nfev: 21
     njev: 21
 hess_inv: <2672x2672 LbfgsInvHessProduct with dtype=float64>
```

I change parameters and always have the errors. I know that it comes from my circular constraint effector, cause without it works perfectly. I am pretty sure there is no error in my class CircularConstraint, since it is an adaption of a previous constraint effector ( [`BarrierElasticity`](https://github.com/DamCB/tyssue/blob/master/tyssue/dynamics/effectors.py#L596)). 

I'll ask google, and this is probably the most useful answer : https://stackoverflow.com/questions/34663539/scipy-optimize-fmin-l-bfgs-b-returns-abnormal-termination-in-lnsrch
It say it may come from : 
    - issue with the descent gradient direction, I am telling it to go down the hill by going up the hill, meaning my gradient does not match my function. -> I don't think this is what append here
    - my function is too flat -> that may be the case, but in that case why there is no errors with `BarrierElasticity` in 2.5/3D?

If anyone has any idea where the error might have come from or how to fix it :)

I attached a small example just int case. 
[minimal_example_error.zip](https://github.com/DamCB/tyssue/files/13218913/minimal_example_error.zip)

Thanks, 

Sophie

**Comment by sniffo** [comment]
This is probably not helpful but I have also recently tried using a similar circular constraint on a circular tissue with the Euler solver. My issue was consistently running into overflow errors. But since my work mostly involves hard boundaries, I was able to get away with simply taking the tangential gradient components for the boundary vertices. I have active, non-conservative forces in my model so I have to use the EulerSolver. What parameters are you using?

**Comment by sophietheis** [comment]
I use the quasistatic solver, that use scipy.optimize.minimize with l-bfgs-b method, with default values. First, I only tried to change `gtol` and `ftol` paramaters as I only use those before. 
I found a solution that seems to work. I increase the parameter `maxls` to 30 (default value is 20) when I call `scipy.optimize.minimize(method="l-bfgs-b)`. This parameter define the maximum number of line search steps per iteration. 
Now everything seems to be working as expected.
